tests/e2e/specs/in-game/one-offs/5_fives.spec.js

```diff
@@ -350,8 +350,11 @@ describe('FIVES', () => {
           deck: [],
         });
       });
+    });
+
+    describe('Illegal FIVES', () => {
 
-      it('Attempts to play 5 with an empty deck', () => {
+      it('Cannot to play 5 with an empty deck', () => {
         cy.skipOnGameStateApi();
```

Comment:
Still can't run this test in gamestate api because it requires passing. I'll add this to the list of specs to unskip in that issue

Reply:
Actually it looks like we might be missing an issue to enable passing? I'll investigate